workflows/src/main/kotlin/com/r3/corda/lib/tokens/workflows/internal/flows/distribution/DistributionUtilities.kt

```diff
@@ -42,3 +49,39 @@ fun getDistributionRecord(serviceHub: ServiceHub, linearId: UniqueIdentifier, pa
 fun hasDistributionRecord(serviceHub: ServiceHub, linearId: UniqueIdentifier, party: Party): Boolean {
     return getDistributionRecord(serviceHub, linearId, party) != null
 }
+
+/**
+ * Utility function to persist a new entity pertaining to a distribution record.
+ * TODO: Add some error handling.
+ * TODO: Don't duplicate pairs of linearId and party.
+ */
+fun ServiceHub.addPartyToDistributionList(party: Party, linearId: UniqueIdentifier) {
```

Comment:
Is all the code below duplicated? I am pretty sure i saw it already but i don't see that you removed it from different module.

Reply:
you are right, i forgot to delete the old file!